scripts/kmodtool

```diff
@@ -397,7 +397,7 @@ print_rpmtemplate ()
 	# and print it and some other required stuff as macro
 	print_rpmtemplate_header
 
-	# now print the packages itselfs
+	# now print the packages itself
```

Comment:
This is still wrong (plural vs singular mismatch). It should either be:
"now print the package itself"

or
"now print the packages themselves"

or
"now print the packages"

It looks like it is printing multiple packages, so I'd probably go with the last one.

Reply:
Done!